document/document_en.txt

```diff
@@ -14,50 +14,50 @@ Spin detection:
 	- The Spin is a Mini if it only has 2 points; the piece is one of S, Z, J, L, T; AND the line clear did not clear the entire piece.
 
 Attack system:
-	Regular line clears (clearing <=3 lines):
+	Normal line clears (1 to 3 lines):
 		Sends (lines cleared -0.5) attack
 	Special line clears:
 		Spin sends (lines cleared x2) attack,
 		  - B2B sends extra 1/1/2/4/8 (for Spin Single/Double/Triple/Techrash/Techrash+ respectively)
 		  - B2B2B sends (lines cleared x0.5), and +1 extra blocking
-		  - If it's a Mini, x0.25
+		  - Minis reduces the attack to 25% (x0.25 multiplier)
 		Non-Spin Techrash/Techrash+ sends (lines cleared) attack,
-		  - B2B sends extra 1 attack,
-		  - B2B2B sends 50% more attack and +1 extra blocking.
+		  - B2B sends 1 additional line
+		  - B2B2B will have an attack boost of 50% and +1 blocking
```

Comment:
still "extra blocking" please for consistency across the document. Or alternatively, change every other instance of "extra blocking" to just "blocking".